packages/server/server.py

```
@@ -0,0 +1,249 @@
+        return flask.jsonify(error=True, message="inavlid search_query"), 400
+    scheme = "https"
+    netloc = "www.youtube.com"
+    path = "/results"
+    params = None
+    queries = {
+        "search_query": f'{search_query} "Topic"',
+    }
+    query = urllib.parse.urlencode(queries)
+    fragment = None
+    url = urllib.parse.urlunparse((scheme, netloc, path, params, query, fragment))
+    response = requests.get(url)
```

Ruff (0.12.2)

Async functions should not call blocking HTTP methods

(ASYNC210)